MobileWallet/Common/Theme.swift

```diff
@@ -117,6 +117,10 @@ struct Colors: Loopable {
     let transactionCellValuePositiveText = UIColor(named: "TransactionCellValuePositiveText")
     let transactionSmallSubheadingLabel = UIColor(named: "SmallSubheading")
 
+    // Sending Tari
+    let sendingTariTitle = UIColor(named: "SendingTariTitleTextBlack")
+    let sendingTariBackground = UIColor(named: "SendingTariBackgroundWhite")
```

Comment:
Same, don't include the color name in the resource name

Reply:
DOne.